dassh/material.py

```diff
@@ -69,18 +69,26 @@ class Material(LoggedClass):
         - Thermal conductivity (k): W/m-K
 
     """
-
+    MATERIAL_LBH = {
+            'lead': Lead,
+            'bismuth': Bismuth,
+            'lbe': LBE
+        }
+    LBH15_PROPERTIES = ['rho', 'cp', 'mu', 'k']
+    PROP_NAME = dict(zip(['density', 'heat_capacity', 'viscosity', 'thermal_conductivity'], LBH15_PROPERTIES)) 
+    
     def __init__(self, name, temperature=298.15, from_file=None,
-                 coeff_dict=None, use_lbh15 = False):
+                 coeff_dict=None, use_lbh15 = False, lbh15_correlations = None):
         LoggedClass.__init__(self, 0, f'dassh.Material.{name}')
         self.name = name
         self.temperature = temperature
-        self.use_lbh15 = use_lbh15
         # Read data into instance; use again to update properties later
         if from_file:
             self.read_from_file(from_file)
         elif coeff_dict:
             self._define_from_coeff(coeff_dict)
+        elif self.name in Material.MATERIAL_LBH.keys() and use_lbh15:
```

Comment:
I'd suggest to check only the value of the flag `use_lbh15`. Then, in the corresponding if statement branch I'd suggest to adopt the `try-except` structure adopted in the last if statement branch. This implies to change the implementation of the method `_define_from_lbh15()` by raising an error in case the selected liquid metal is not managed by lbh15.